plugins/catalog-react/src/components/EntityAutocompletePicker/EntityAutocompletePicker.tsx

```diff
@@ -130,7 +130,7 @@ export function EntityAutocompletePicker<
     <Box pb={1} pt={1}>
       <Typography variant="button" component="label">
         {label}
-        <Autocomplete
+        <Autocomplete<string, true>
```

Comment:
This is unfortunate. Didn't dig all that deeply into what causes this, but what I've written here is the actual type arguments that used to be inferred but the ancient (since several years no longer maintained) lab package seems to take issue with newest TS so I help it to infer what should be here.

Ping @Rugvip for visibility